### PR TITLE
AT-UNIVERSITY-UI Admin / Dashboard Module Migration

### PR DESCRIPTION
**Pull request by at-internship**
AT-UNIVERSITY-UI Admin / Dashboard Module Migration

### CODE REVIEW

**Review by fasaga — APPROVED** [review]
I run it locally and looks good to me

File: src/controllers/admin.controller.js [hunk 1]
```diff
@@ -1,53 +1,71 @@
...
+// AT-UNIVERSITY - Admin - Render Course List
+adminCtrl.renderCourseList = async(req, res) => {
+    try {
+        const responseCourses = await universityServiceAPI.getAllCourses();
+        console.log("---> adminCtrl.renderCourseList.getAllCourses");
+        //console.log(responseCourses.data);
+        const courses = responseCourses.data;
+        res.render("admin/course/index", { courses });
+    } catch (err) {
+        console.error(err.message);
```
Comment (raaul-at) [inline src/controllers/admin.controller.js:20]: Is this error handling necessary?
I reviewed the implementation of `getAllCourses` and you are handling errors there, please double check your implementation.

**Review by raaul-at — CHANGES_REQUESTED** [review]

File: src/controllers/admin.controller.js (hunk 1 above)
Comment (at-internship) [reply src/controllers/admin.controller.js:20]: oh yes, imagine if universityServiceAPI.getAllCourses(); return a timeout

Comment (raaul-at) [reply src/controllers/admin.controller.js:20]: Great, thank you for double check 👌

**Review by raaul-at — APPROVED** [review]
👍

**Pull request merged**